dstep/translator/Context.d

```diff
@@ -204,3 +225,66 @@ bool isGlobal(Cursor cursor)
 {
     return cursor.semanticParent.kind == CXCursorKind.CXCursor_TranslationUnit;
 }
+
+/**
+  * The collectTypeNames function scans the whole AST of the translation unit and produces
+  * a set of the type names in global scope.
+  *
+  * The type names are required for parsing C code (e.g. macro definition bodies),
+  * as C grammar isn't context free.
+  */
```

Comment:
* Same here with the alignment of `*`
* I still don't understand how this function is used